docs/roman.rst

```diff
@@ -12,35 +15,59 @@ Wide Field Instrument (WFI)
 ===========================
 
 
-.. figure:: ./roman_figures/stpsf-roman_page_header.png
+.. figure:: ./roman_figures/stpsf_roman_page_header.png
    :align: center
    :alt: Sample PSFs for the filters in the Roman WFI.
 
-   Sample PSFs for the filters in the Roman WFI on SCA01. Angular
+   Sample PSFs for the filters in the Roman WFI on detector WFI01. Angular
    scale in arcseconds, log-scaled intensity. Note that the prism and
    grism PSFs shown here are monochromatic.
 
-The WFI model is based on the `Cycle 9 instrument reference information <https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html>`_ from the Roman team at Goddard Space Flight Center (GSFC). The reported jitter for the Roman observatory is 0.012 arcsec per axis, per `GSFC <https://roman.ipac.caltech.edu/sims/Param_db.html#telescope>`_.
+The WFI model is based on the
+`Cycle 10 instrument reference information <https://roman.gsfc.nasa.gov/science/Roman_Reference_Information.html>`_
```

Comment:
It looks like GSFC removed the spreadsheet at this link. (They previously had a copy of the spreadsheet we used during the Cycle 9 update here.) Should we keep the link? Go somewhere else?

Reply:
They removed the file from the website a while ago and it's not available anywhere other than on request, so let's remove this particular link.